app/datadog-init.tsx

```diff
@@ -0,0 +1,25 @@
+"use client";
+
+import { datadogRum } from "@datadog/browser-rum";
+
+
+console.log("Datadog Init", process.env.NEXT_PUBLIC_DATADOG_RUM_APPLICATION_ID, process.env.NEXT_PUBLIC_DATADOG_RUM_CLIENT_TOKEN);
+
+datadogRum.init({
+  applicationId: process.env.NEXT_PUBLIC_DATADOG_RUM_APPLICATION_ID || "",
+  clientToken: process.env.NEXT_PUBLIC_DATADOG_RUM_CLIENT_TOKEN || "",
+  site: "datadoghq.com",
+  service: "avail-bridge-ui",
+  env: process.env.NEXT_PUBLIC_ENVIRONMENT || "staging",
+  version: "1.0.0",
+  sessionSampleRate: 100,
+  sessionReplaySampleRate: 20,
```

Comment:
We may reduce the `sessionSampleRate` and `sessionReplaySampleRate` in future PRs in order to reduce costs.
Let's keep track of it.